src/main/java/com/gmail/goosius/siegewar/command/SiegeWarCommand.java

```diff
@@ -67,4 +83,36 @@ private void parseSiegeWarNationCommand(Player player, String[] remFirstArg) {
 		}
 	}
 
+	private void parseSiegeWarHudCommand(Player player, String[] args) {
+		try {
+			if (args.length == 0) {
+				player.sendMessage(ChatTools.formatTitle("/siegewar"));
+				player.sendMessage(ChatTools.formatCommand("Eg", "/sw nation", "refund", Translation.of("nation_help_11")));
+				player.sendMessage(ChatTools.formatCommand("Eg", "/sw hud", "[town]", ""));
+			} else {
+				List<String> townsBeingSieged = getOngoingSieges();
+				Town town = TownyUniverse.getInstance().getTown(args[0]);
+				if (town == null) 
+					throw new TownyException(Translation.of("msg_err_town_not_registered", args[0]));
+
+				if (!townsBeingSieged.contains(town.getName())) {
+					Messaging.sendErrorMsg(player, Translation.of("msg_err_not_being_sieged", town.getName()));
+					return;
+				}

```

Comment:
Can be condensed to:
```java
				if (!SiegeController.getSiegedTowns().contains(town)) {
					Messaging.sendErrorMsg(player, Translation.of("msg_err_not_being_sieged", town.getName()));
					return;
				}

Reply:
Both if's or just the bottom one? An invalid town name would make town null

Reply:
Ah my bad, you are correct.